### DISCUSSION TITLE
Writing using XSharedPreferences

### DISCUSSION THREAD

**Discussion by ItsMadruga** [LSPosed/LSPosed#2910] [New Release]
I liked that you fixed the XSharedPreferences reading problems, but I was developing my module, and I hit a wall, I needed to make it save a boolean in the sharedPref, I tried using `edit()` as it is common in SharedPreferences, but it only returned a error `read-only implementation`, I looked at the Xposed API and read that it only supported reading and not writing, could you make this possible? ;)

**Comment by yujincheng08** [comment]
It's expected to be read only.

**Reply by ItsMadruga** [reply]
The problem itself occurs when I need to update the boolean, there is no `context `to call `SharedPreferences`, so I would have to resort to `XSharedPreferences`, which **only supports reading**

**Reply by yujincheng08** [reply]
As said, expected.
